components/layout/section/section.module.css

```diff
@@ -2,7 +2,7 @@
   display: flex;
```

Comment:
consider moving styles from here to the sx prop. then, i think, we won't have to worry about overrides (!important)

Reply:
`section.js` isn't actually calling any styles from this file so i think we could delete it altogether?